cmd/verify/network/cmd.go

```diff
@@ -153,8 +152,19 @@ func run(cmd *cobra.Command, _ []string) {
 			os.Exit(1)
 		}
 	} else if !cmd.Flags().Changed(statusOnlyFlag) {
-		r.Reporter.Errorf("%s is required", roleArnFlag)
-		os.Exit(1)
+		if cmd.Flags().Changed(clusterFlag) {
+			if cluster == nil {
+				cluster = r.FetchCluster()
+			}
+			if args.roleArn = cluster.AWS().STS().RoleARN(); args.roleArn == "" {
+				r.Reporter.Errorf("Unable to get role ARN from non-STS cluster '%s'.",
+					cluster.Name())
```

Comment:
FWIW its best to return `cluster.ID()` then `cluster.Name()` this is common pattern in OCM. 

Just makes our life easier for debugging issues when people report them to search by ID than Name

Reply:
i changed to `cluster.ID()` but should both name and id be output? do users typically specify id or name as a cli argument?